Package.swift

```diff
@@ -3,7 +3,7 @@ import PackageDescription
 
 let package = Package(
     name: "Zephyr",
-    platforms: [.iOS(.v11), .tvOS(.v11)],
+    platforms: [.iOS(.v11), .tvOS(.v11), .watchOS("9.0")],
```

Comment:
Why not `v.9`?

Reply:
Unfortunately `.v9` shortcut is not available. Also `.v15` or `.v16` for iOS do not exists. I do not know why Apple does not offer them any more.